src/components/Modal/Strategy.vue

```diff
@@ -39,6 +40,16 @@ function handleSubmit() {
   emit('close');
 }
 
+function handleClick(strategy) {
+  const { params } = strategy.examples[0].strategy;
+  const hasStrategy = !isEmpty(params);
```

Comment:
You could simplify and do like this
```
const params = strategy.examples[0]?.strategy?.params || defaultParams

Reply:
@KennethAshley Some strategies dont require any params, it make sense to display empty object in this case so i wouldn't bother doing this extra checks. I will return false only when there isn't any example.

Reply:
@bonustrack awesome. thanks for the reply. will update